src/components/views/Group.js

```diff
@@ -3,93 +3,172 @@ import Button from '../shared/Button'
 import ActionSlide from '../shared/ActionSlide'
 import Task from '../cards/Task'
 import Member from '../cards/Member'
-import styled from 'styled-components'
 import ContentWrapper from '../../styles/ContentWrapper'
 import InfoCard from '../cards/InfoCard'
 import Switcher from '../shared/Switcher'
-import { graphql, gql, withApollo } from 'react-apollo'
+import { gql, withApollo } from 'react-apollo'
 
 class Group extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
       openMenu: false,
-      active: 'Members',
+      active: 'Tasks',
       name: null,
-      tasks: []
+      settings: false,
+      tasks: [],
+      completed: false
     }
-    this.handleButtonClick = this.handleButtonClick.bind(this)
+
     this.handleSwitcherClick = this.handleSwitcherClick.bind(this)
+    this.handleGroupUpdate = this.handleGroupUpdate.bind(this)
```

Comment:
Part of the class fields discussion will include getting rid of `this.methodName.bind(this)` as well 😁